include/Core/Request/LogListAction.php

```diff
@@ -27,7 +27,20 @@ public function load() {
 	 * @since 1.6
 	 */
 	public function view_log_message() {
-		if ( current_user_can( 'manage_options' ) ) {
+
+		$user_capability = 'manage_options';
+
+		/**
+		 * Filters the User capability to View Email Log content.
+		 *
+		 * Refer User Capabilities at
+		 * @link https://codex.wordpress.org/Roles_and_Capabilities#Capabilities
+		 *
+		 * @param string $user_capability User capability to view Log content.
+		 */
+		$view_content_user_capability = apply_filters( 'view_content_user_capability', $user_capability );
```

Comment:
Couple of things.

- Let's prefix the filter with `el_`, like the rest of the actions and filters.
- Let's add a `@since` attribute to the phpdoc
- Let's remove two variables and replace the entire thing with just the following. This way it will be more readable.

```
if ( current_user_can( apply_filters( 'el_view_email_log_capability', 'manage_options' ) ) ) {
```